releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,7 +1,19 @@
-## Microsoft.Extensions.Configuration.AzureAppConfiguration
-### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
+# Microsoft.Extensions.Configuration.AzureAppConfiguration
+[Source code][source_code] | [Package (NuGet)][package]
 
-### 7.0.0 - November 21, 2023
+## 7.1.0-preview - January 19, 2024
+### Replica Auto Discovery
+* For App Configuration stores with geo-replication enabled, the provider will now automatically discover any additional replicas and attempt to connect to them when it fails to connect to user-provided endpoints in code. This capability allows applications to leverage geo-replication for enhanced resiliency without code change and redeployment. Replica discovery is enabled by default and can be disabled by setting the new `AzureAppConfigurationOptions.ReplicaDiscoveryEnabled` property equal to `false`. [#442](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/442).
```

Comment:
I understand we want to emphasize this but having a separate section is a bit confusing what it is about. Does it make sense to nest it under enhancements but have a bolded "Replica auto discovery" at the beginning?

Reply:
It makes sense to have under enhancements, and yea that seems to be the best way to add a title to a bullet point that I can find. The other thought I had was to make the `Enhancements` section say `Other Enhancements` instead.